scalecube-configuration/src/test/java/io/scalecube/configuration/fixtures/InMemoryConfigurationServiceFixture.java

```diff
@@ -103,7 +103,7 @@ private OrganizationService mockOrganization() {
     Builder organization1InfoBuilder =
         OrganizationInfo.builder()
             .id(organizationId1)
-            .name("Test Organization 1")
+            .name("Test_Organization_Name_1")
```

Comment:
What is the purpose of this change?

Reply:
It's just for fast intuitive visibility of relation to name.

Reply:
Not sure that understood it. Could you please explain it in details?